<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>spotify-playlist-qt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QGroupBox" name="groupBoxCurrentSong">
      <property name="title">
       <string>SongName</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonBack">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPlay">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonNext">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderSongProgress">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBoxSearch">
          <property name="title">
           <string>Buscar</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLineEdit" name="lineEditSearch"/>
             </item>
             <item>
              <widget class="QTableWidget" name="tableWidgetSearch">
               <column>
                <property name="text">
                 <string>Título</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Artista</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxPlaylist">
          <property name="title">
           <string>Playslists</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QListWidget" name="listWidgetPlaylist"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxSongs">
        <property name="title">
         <string>Músicas</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTableWidget" name="tableWidgetTracks">
           <column>
            <property name="text">
             <string>Título</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Artista</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Album</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tamanho</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1195</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMusicas">
    <property name="title">
     <string>Músicas</string>
    </property>
    <addaction name="actionUser"/>
    <addaction name="actionConnectAPI"/>
    <addaction name="actionNewPlaylist"/>
    <addaction name="actionRemove"/>
   </widget>
   <addaction name="menuMusicas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUser">
   <property name="text">
    <string>Usuário</string>
   </property>
  </action>
  <action name="actionNewPlaylist">
   <property name="text">
    <string>Nova Playlist</string>
   </property>
  </action>
  <action name="actionConnectAPI">
   <property name="text">
    <string>Conectar API</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remover</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
